frontend/server/k8s-helper.ts

```diff
@@ -127,3 +127,36 @@ export function getPodLogs(podName: string): Promise<string> {
       (error: any) => { throw new Error(JSON.stringify(error.body)); }
     );
 }
+
+/**
+* Creates a CRD (Customer Resource Definition) file. This CRD file can be used
+* by CRD controller to create new Tensorboard Pod.
+*/
+export async function newTensorboardCRD(logdir: string): Promise<void> {
+  const crd = {
```

Comment:
is it possible to use k8s js client?
https://github.com/kubernetes-client/javascript

hard coding template is considered not a good practice and we should avoid if possible :)

Reply:
Sure. Let me figure out how to use k8s js client and update my change. If you can help me with some similar examples, that will be really helpful to me :)